src/common/utils.ts

```diff
@@ -0,0 +1,73 @@
+import { SceneDataTransformer, SceneObject, SceneObjectState, SceneQueryRunner, VizPanel } from '@grafana/scenes';
+import { lte } from 'semver';
+import { PanelModel } from './dashboard';
+import { config } from '@grafana/runtime';
+
+export async function refreshPanelsInDashboard() {
+  if (isGrafanaLte10()) {
+    refreshAngularDashboard();
+  } else {
+    const panels = await getAllDashboardPanels();
+    panels.forEach((panel) => {
+      const queryRunner = getQueryRunnerFor(panel);
+      queryRunner?.runQueries();
+    });
+  }
+}
+
+async function getAllDashboardPanels(): Promise<VizPanel[]> {
+  const sceneContext: SceneObject<SceneObjectState> = (window as any).__grafanaSceneContext;
+  if (sceneContext) {
+    try {
+      // alarm panel fails to load if sceneGraph is imported at the top level in 10, so it needs to be conditionally imported
+      const scenes = await import(/* webpackMode: "eager" */ '@grafana/scenes');
+      return scenes.sceneGraph.findAllObjects(
+        sceneContext,
+        (obj) => obj?.state?.key?.includes('panel') ?? false
```

Comment:
had to duck type these and some others (SceneQueryRunner, SceneDataTransformer) since `instanceof` doesn't work